qemu/tests/rng_hotplug.py

```diff
@@ -28,9 +28,10 @@ def run(test, params, env):
 
     def get_rng_id(vm):
         device_list = []
+        rng_driver = "virtio-rng-pci"
```

Comment:
This line is unnecessary since we usually do this in the cfg file which you already have done.

Reply:
Instead we need to get the value of rng_driver as follows:

rng_driver = params.get("rng_driver")

Reply:
Thank you for pointing, Nini. Done.